packages/n/nushell/package.yml

```diff
@@ -0,0 +1,19 @@
+name       : nushell
+version    : 0.87.1
+release    : 1
+homepage   : https://www.nushell.sh/
+networking : yes
+source     :
+    - https://github.com/nushell/nushell/archive/refs/tags/0.87.1.tar.gz : 92087ff56c98acb86dc14e9566748c0f470ad5f13277dd62bda878146535fa83
+license    : MIT
+component  : system.utils
+summary    : A new type of shell.
+description: |
+    Nu takes cues from a lot of familiar territory traditional shells like bash, object based shells like PowerShell, gradually typed languages like TypeScript, functional programming, systems programming, and more. But rather than trying to be a jack of all trades, Nu focuses its energy on doing a few things well
+builddeps  :
+    - rust
+    - cargo
+install    : |
```

Comment:
Please take a look at `amdgpu_top` to see how a rust project should be built. You need to split it up into three phases: fetching dependencies (during `setup`), building it (`build`), and finally the install. Make sure you use the `--locked` flag so that it's reproducible.